### ISSUE TITLE
Function to set the window title

### ISSUE THREAD

**Issue by nomeata** [ku-fpg/blank-canvas#82]
Currently, the browser tab displaying the blank canvas app says `Blank Canvas`. Which is a lie as soon as the canvas is filled :-).

I propose a function `setTitle :: Text -> Canvas ()` that would, well,  set the the title.

**Comment by RyanGlScott** [comment]
Good idea! Luckily, this kind of thing is super-easy to add to `blank-canvas`.

The trickiest part is adding the JavaScript functionality. You'd need to add something like this to `static/index.html`:

```javascript
function SetTitle(newTitle) { // :: Text -> Canvas ()
    return function(u, c) {
        document.title = newTitle;
        $.kc.reply(u, []); // Return unit tuple
    };
}
```

Once that's done, all you need to do is add a new `SetTitle` constructor to the [`Query`](https://github.com/ku-fpg/blank-canvas/blob/042f99280399e337ea102d2d167999f8653f8ee3/Graphics/Blank/Canvas.hs#L268) datatype, as well as corresponding cases to the [`InstrShow`](https://github.com/ku-fpg/blank-canvas/blob/042f99280399e337ea102d2d167999f8653f8ee3/Graphics/Blank/Canvas.hs#L285) instance and [`parseQueryResult`](https://github.com/ku-fpg/blank-canvas/blob/042f99280399e337ea102d2d167999f8653f8ee3/Graphics/Blank/Canvas.hs#L308).

Then `setTitle` simply becomes:

```haskell
setTitle :: Text -> Canvas ()
setTitle = procedure . Query . SetTitle
```
